assets/stylesheets/main.css

```diff
@@ -5,5 +5,69 @@
     width: 1000px;
     height: 500px;
     border: 4px solid black;
+    border-radius: 20px;
+    background-color: white;
 }
 
+.tools {
+    background-color: #DDDDDD;
+    border: 4px solid #BDBDBD;
+    border-radius: 20px;
+    box-sizing: border-box;
+    width: 100%;
+    height: 100%;
+}
+
+#details {
+    background-color: #DDDDDD;
+    border: 4px solid #BDBDBD;
+    border-radius: 20px;
+    box-sizing: border-box;
+    width: 100%;
+    height: 100%;
+}
+
+#grid {
+    display: grid;
+    width: 75%;
+    margin: auto;
+    background-color: #9D9D9D;
+    grid-template:
+        "title title title"
+        "options options options"
+        "toolbar canvas-cell details";
+    grid-template-columns: 100px 100fr 250px;
```

Comment:
100fr?

Reply:
1fr should suffice shouldn't it

Reply:
You're right, I assumed it worked like percentages lmao

I'll change that